site/content/blog/eth2-dev-chain.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Setting up private Ethereum 2.0 development chain
+date: 2020-03-26T09:00:00-00:00
+image: /images/blog/ethereum20-validator.png
+draft: false
+keywords:
+- ethereum 2.0
+- proof of staking
+- dev
+- lighthouse
+- chainguardian
+description: Learn how to setup eth2 development chain using lighthouse client. 
+---
+
+> Disclaimer: This article is highly technical intended for developers and eth2 enthusiasts.
+
+### Why would I setup private eth2 chain?
+
+If you had any experience with development on eth1, you probably already know answer to this question so 
+feel free to skip ahead.
+
+At the moment, eth2 has a bunch short lived single client testnets and [one long lived](https://prylabs.net/) with
+real stable client testnet still TBD. So, yeah, you can use [Prysmatic's Sapphire Testnet](https://prylabs.net/) but
+beware it will take some time to sync and if you wanna test some validator related operations, you will have to wait
+couple of hours to become active validator. Imagine, waiting couple of hours just to find out your app has bug 
+with validator related operation and now you have to do everything all over.
+
+Our motivation for this setup was [ChainGuardian](https://github.com/nodefactoryio/chainguardian), a desktop validator
```

Comment:
is it a validator `client` really or more of desktop app for validators or something like that?

Reply:
From my experience, if I say or write any else then "desktop validator client" people have no idea what I'm talking about. And it is, as a matter of fact, our core functionality

Reply:
idk, I was referring that technically client is something like lodestar, lighthouse etc.

Reply:
those are beacon nodes. They do have validators but they are cli tools